pkg/enqueue/Doctrine/DoctrineConnectionFactoryFactory.php

```diff
@@ -2,16 +2,16 @@
 
 namespace Enqueue\Doctrine;
 
+use Doctrine\Bundle\DoctrineBundle\Registry;
```

Comment:
You should use `Doctrine\Common\Persistence\ManagerRegistry` instead as noted in the [upgrade doc](https://github.com/doctrine/DoctrineBundle/blob/master/UPGRADE-2.0.md#service-aliases)

Reply:
updated to `ManagerRegistry`